amethyst_audio/src/output.rs

```diff
@@ -75,11 +67,17 @@ impl Output {
     ) -> Result<(), DecoderError> {
         let sink = Sink::new(&self.endpoint);
         for _ in 0..n {
-            sink.append(
-                Decoder::new(Cursor::new(source.clone()))
-                    .map_err(|_| DecoderError)?
-                    .amplify(volume),
-            );
+            if SOUNDS_PLAYING.load(Ordering::Relaxed) < MAX_SOUNDS_PLAYING {
```

Comment:
`Relaxed` is not enough here.

Reply:
So if I'm being completely honest I'm a little fuzzy on which ordering you should be using and for what reason you should be using it.  Relaxed seems fine here though, as it's not the end of the world if we're playing 302 sounds instead of 300.